src/app/utils/permission-levels.js

```diff
@@ -2,6 +2,7 @@ const PERMISSION_LEVELS = {
   READ: 'read',
   WRITE: 'write',
   DELETE: 'delete',
+  ADMIN: 'admin',
```

Comment:
Also, it seems like `ADMIN/DELETE` are the same thing from looking through the code base?

Reply:
@liangyuanruo requested this

Reply:
to refactor `DELETE` to `ADMIN`